package.json

```diff
@@ -150,7 +150,7 @@
 	},
 	"devDependencies": {
 		"@types/node": "^13.1.1",
-		"@types/vscode": "~1.20.0",
+		"@types/vscode": "^1.41.0",
```

Comment:
We need the newer version of `@types/vscode`. Also, this PR using the newer API of `vscode`.

Reply:
If this does not kill compatibility with older versions of VSCode, then okay.

Reply:
No... it's a new API.